Python/Product/Analysis/Interpreter/Ast/AstAnalysisWalker.cs

```diff
@@ -153,37 +153,45 @@ bool includeLocationInfo
                 modName,
                 PythonAnalyzer.ResolvePotentialModuleNames(_module.Name, _filePath, modName, true).ToArray()
             );
-            mod.Imported(_scope.Context);
-            // Ensure child modules have been loaded
-            mod.GetChildrenModules();
-
-            try {
-                for (int i = 0; i < node.Names.Count; ++i) {
-                    if (!onlyImportModules) {
-                        if (node.Names[i].Name == "*") {
-                            foreach (var member in mod.GetMemberNames(_scope.Context)) {
-                                var mem = mod.GetMember(_scope.Context, member) ?? new AstPythonConstant(
+            var interp = _scope.Context as AstPythonInterpreter;
+            if (interp?.BeginNestedImport() ?? true) {
+                mod.Imported(_scope.Context);
```

Comment:
Should this whole block be under a try/finally to ensure EndNestedImport is called?

Reply:
EndNestedImport was removed in the other PR. This one will be re-merged and it will go away here too.